src/main/less/core.less

```diff
@@ -279,3 +279,15 @@ svg:not(:root) {
         }
     }
 }
+
+.no-data-text {
+    font-family: "Helvetica Neue";
+    font-size: 24px;
```

Comment:
Can you have this in `rem`? 
https://engageinteractive.co.uk/blog/em-vs-rem-vs-px

Reply:
Updated